oidc-authenticator-configuration/src/main/java/org/xwiki/contrib/oidc/auth/internal/store/DefaultOIDCClientConfigurationStore.java

```diff
@@ -116,8 +120,15 @@ public OIDCClientConfiguration getOIDCClientConfiguration(String name) throws XW
         OIDCClientConfiguration configuration = null;
         XWikiDocument configurationDocument = getOIDCClientConfigurationDocument(name);
         if (configurationDocument != null) {
-            configuration =
-                new OIDCClientConfiguration(configurationDocument.getXObject(OIDCClientConfiguration.CLASS_REFERENCE));
+            Optional<BaseObject> configObj = configurationDocument.getXObjects(OIDCClientConfiguration.CLASS_REFERENCE)
```

Comment:
Would probably make more sense to use `XWikiDocument#getXObject(DocumentReference classReference, String key, String value)`.

Reply:
I yes, didn't think about this API. Should be fixed